style.css

```diff
@@ -0,0 +1,104 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+html {
```

Comment:
- [ ] I think it would be better if you inform Code Reviewer by writing a descriptive description on your PR. Kindly add some description in PR. 😉

It is not allowed for example this one:
![Screen Shot 2021-11-23 at 7 02 51 PM](https://user-images.githubusercontent.com/69011963/143252664-73e1ddda-93a4-4446-8584-96be5fc899c8.png)

Reply:
Hi @Rustamxon7 thank you for your time reviewing my project. I had this issue whenever I try to change the base branch it says "There are no new commits between base branch 'main' and head branch 'main' " so should I create a new pull request or do you have any idea on how to get around this issue?